app/models/collection_change_set.rb

```diff
@@ -44,6 +44,22 @@ def changed_participants
       removed_reviewers).uniq
   end
 
+  def email_depositors_status_changed_changed?
```

Comment:
changed_changed?

Reply:
Unfortunately. The field is called `email_depositors_status_changed` so checking whether this field has changed appends `_changed?`.